src/index.ts

```diff
@@ -177,16 +186,13 @@ const config = Config.fromFile("./config.toml");
     );
   }, pollIntervalMillis);
 
-  try {
-    if (bitcoinWallet) {
-      await bitcoinWallet.refreshUtxo();
-      setInterval(() => {
-        bitcoinWallet.refreshUtxo();
-      }, 60000);
-    }
-  } catch (e) {
-    logger.error("Failed to refresh UTXO:", e.message);
-    console.log(e.message);
+  if (bitcoinWallet) {
+    setTimeout(async () => {
```

Comment:
❓ I don't really understand why we need `setTimeout(...)` here, but maybe I am not understanding it correctly. According to my understanding the callback of `setTimeout` is called after the specified timeout (10ms); as specified here: http://www.java2s.com/Tutorials/Javascript/Node.js_Tutorial/0270__Node.js_setTimeout_setInterval.htm
I think the purpose of this code piece is `await refreshUtxos(bitcoinWallet)` for the first time, and then start the interval. We do we need `setTimeout` here? I found it easier to understand before.